kornia/geometry/warp/homography_warper.py

```diff
@@ -94,6 +119,55 @@ def homography_warp(patch_src: torch.Tensor,
                          align_corners=align_corners)
 
 
+def homography_warp3d(patch_src: torch.Tensor,
+                      src_homo_dst: torch.Tensor,
+                      dsize: Tuple[int, int, int],
+                      mode: str = 'bilinear',
+                      padding_mode: str = 'zeros',
+                      align_corners: bool = False,
+                      normalized_coordinates: bool = True) -> torch.Tensor:
+    r"""Function that warps image patchs or tensors by homographies.
```

Comment:
not sure that we have to call this `homography_warp3d` or `affine_warp3d` since the transformation it's not an homography anymore but a 3D affine transformation.

Reply:
/cc @ducha-aiki

Reply:
affine_warp3d or warp_affine3d